tsl/test/expected/continuous_aggs_refresh.out

```diff
@@ -423,3 +423,37 @@ SELECT * FROM weekly_temp_without_data;
  Sun May 03 17:00:00 2020 PDT |      3 |               19
 (8 rows)
 
+-- This should fail since we do not allow refreshing inside a
+-- transaction, not even as part of CREATE MATERIALIZED VIEW.
+\set ON_ERROR_STOP 0
+DO
+LANGUAGE PLPGSQL
+$$ DECLARE ts_version TEXT;
+
+BEGIN
+CREATE MATERIALIZED VIEW weekly_conditions
+WITH (timescaledb.continuous,
+      timescaledb.materialized_only=true)
+AS
+SELECT time_bucket('7 days', time) AS day, device, avg(temp) AS avg_temp
+FROM conditions
+GROUP BY 1,2 WITH DATA;
+END $$;
+ERROR:  CREATE MATERIALIZED VIEW ... WITH DATA cannot be executed from a function
+\set ON_ERROR_STOP 1
+-- This should not fail since we do not refresh the continuous
+-- aggregate.
+DO
+LANGUAGE PLPGSQL
+$$ DECLARE ts_version TEXT;
```

Comment:
Another declare block and variable that is never used.

Reply:
Removed.